src/Manifest.cc

```diff
@@ -420,23 +420,32 @@ validateDepName(const StringRef name) {
     throw PoacError("dependency name must end with an alphanumeric character");
   }
 
-  // Only alphanumeric characters, `-`, `_`, and `/` are allowed.
+  // Only alphanumeric characters, `-`, `_`, `/` and `.` if surrounded by
+  // numbers are allowed.
```

Comment:
I would check if these are only the characters for this pass and check if dots are only surrounded by numbers in the next pass.